_includes/tutorials/udf/ksql/markup/dev/continuous-select.adoc

```diff
@@ -0,0 +1,5 @@
+Since the output looks right, the next step is to make the query continuous. Issue the following to create a new stream that is continously populated by its query:
```

Comment:
```suggestion
Since the output looks right, the next step is to make the query continuous. Issue the following to create a new stream that is continuously populated by its query:
```